src/shared/ui/Tabs/TabsRoot.tsx

```diff
@@ -0,0 +1,32 @@
+'use client';
+import { Children, isValidElement, ReactElement, useMemo, useState } from 'react';
+
+import { TabsContext } from './Tabs.context';
+import { TabsNav } from './TabsNav';
+
+type Props = {
+  children: React.ReactNode;
+  defaultIndex?: number;
+};
+
+export function TabsRoot({ children, defaultIndex = 0 }: Props) {
```

Comment:
`...props`를 추가해서 외부에서 전달된 추가적인 속성을 컴포넌트에 넘기도록 하는 것도 좋을 것 같아요.

```suggestion
export function TabsRoot({ children, defaultIndex = 0, ...props }: Props) {
```